R/timeseriessummary-data.R

```diff
@@ -159,6 +160,7 @@ parseTSSRatingCurves <- function(reportData, timezone){
     curves[['applicablePeriods']] <- lapply(curves[['applicablePeriods']], function(p){
       p[['startTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['startTime']], timezone))
       p[['endTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['endTime']], timezone))
+      p <- p[order(p[['startTime']]),]
```

Comment:
I think this sort will only make the order of the applicable periods of the current curve be in order, while the order of the curves themselves may not be. Each curve has a `startOfPeriod` parameter that may also need to be used for sorting outside of this lapply in order to sort the curves themselves.